CSS/style.css

```diff
@@ -115,8 +116,40 @@ footer {
   margin-left: 10px;
 }
 
-.print {
+.delicious {
+  font-family: 'Yellowtail', cursive;;
+  font-size: 38px;
+  display: block;
+  float: left;
```

Comment:
Display: block is the default for h* elements. Remove the float: left if you place it within an inline or inline-block element.